harpy/_validations.py

```
@@ -300,82 +301,92 @@ def validate_demuxschema(infile:str, return_len: bool = False) -> None | int:
-            err += f"- start position: [bold yellow]{reg[1]}[/bold yellow]\n"
-        if reg[2] > contigs[reg[0]]:
-            err += f"- end position: [bold yellow]{reg[2]}[/bold yellow]"
-        if err != "":
-            print_error("region out of bounds", f"Components of the input region [yellow bold]{regioninput}[/yellow bold] were not found in [blue]{genome}[/blue]:\n" + err)
-            sys.exit(1)
-        return "region"
+        return contigs
+
+    try:
+        # is an int
+        region = int(regioninput)
```

Ruff (0.8.2)

Local variable `region` is assigned to but never used

Remove assignment to unused variable `region`

(F841)